server/lib/search.js

```diff
@@ -59,46 +60,37 @@ export const searchCollectivesByEmail = async (email, user, offset = 0, limit =
   return [collectives, get(collectives[0], 'dataValues.__total__', 0)];
 };
 
-/**
- * Turn a search string into a TS vector using 'OR' operator.
- *
- * Ex: "open potatoes" => "open|potatoes"
- */
-const searchTermToTsVector = term => {
-  return term.replace(/\s+/g, '|');
-};
-
-/**
- * Trim leading/trailing spaces and remove multiple spaces from the string
- */
-const trimSearchTerm = term => {
-  return term?.trim().replace(/\s+/g, ' ');
-};
-
-/**
- * Removes special ILIKE characters like `%
- */
-const sanitizeSearchTermForILike = term => {
-  return term.replace(/(_|%|\\)/g, '\\$1');
-};
-
```

Comment:
I don't get why we're moving these. It makes sense to me to have helpers called **search**TermToTsVector, trim**Search**Term, or sanitize**Search**TermForILike (that are purely dedicated to help with search) exported from `lib/search`. If we put all shared code in `lib/utils` we're just going to end up with a gigantic file.

Reply:
That makes sense. I thought somehow that since we are using these multiple times they should go to `utils` but now when you point out that these are all related specifically for search, it makes sense.